# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 7398bc391379
# Full Hash: 7398bc391379a76e4bdde7dd13b0410ea7137ad7
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2021-08-18 15:53:06
# Regressor Bug: 1726063
# File Overlap Count: 2
# Description:
#   Bug 1726063 - Remove option to use BasicLayerManager. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D122796
# ==============================================================================

diff -r f62cad8b2ebd -r 7398bc391379 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Tue Aug 17 21:35:35 2021 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue Aug 17 23:19:44 2021 +0000
@@ -4836,11 +4836,6 @@
   value: true
   mirror: always
 
-- name: gfx.basic_layer_manager.force_enabled
-  type: RelaxedAtomicBool
-  value: false
-  mirror: always
-
 #  Whether to enable LayerScope tool and default listening port.
 - name: gfx.layerscope.enabled
   type: RelaxedAtomicBool